onload.js

```diff
@@ -4,4 +4,4 @@ Module['CONTACT_PROCESSED_CALLBACK_SIGNATURE'] = 'iiii';
 Module['INTERNAL_TICK_CALLBACK_SIGNATURE'] = 'vif';
 
 // Reassign global Ammo to the loaded module:
-this['Ammo'] = Module;
+globalThis['Ammo'] = Module;
```

Comment:
Oh sorry I didn't realize I included this in this PR.  It was added to support loading ammojs in an es module as there is no `this` defined at the top level so it would throw. And `globalThis` is supported everywhere in all major browsers/runtimes. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

I can remove it for this PR if you would like.

Reply:
Yes, thanks, let's do a separate PR for that change. I think it makes sense on its own.

Reply:
Ok I removed everything except the ammo.idl changes. It wasn't clear to me if I should include the builds but it makes sense that you would want to review it and build it yourself before publishing anyways.